sn_node/src/bin/safenode/main.rs

```diff
@@ -86,6 +86,11 @@ struct Opt {
     /// Only applies when --log-dir is also set to output logs to file.
     #[clap(long)]
     json_log_output: bool,
+
+    /// Renew secret key (file) that is used for our peer ID. If the `secret-key` file exists
```

Comment:
Minor style point. I would prefer to keep the use of the backtick notation out of the UI text because it doesn't get parsed in the terminal.